libqtile/widget/volume.py

```diff
@@ -71,7 +71,18 @@ class Volume(base._TextBox):
         ("volume_app", None, "App to control volume"),
         ("volume_up_command", None, "Volume up command"),
         ("volume_down_command", None, "Volume down command"),
-        ("get_volume_command", None, "Command to get the current volume"),
+        (
+            "get_volume_command",
+            None,
+            "Command to get the current volume."
+            "The expected output should include 1-3 numbers and a ``%`` sign.",
+        ),
+        ("check_mute_command", None, "Command to check mute status"),
+        (
+            "check_mute_string",
+            "[off]",
+            "String to look for to check muted status in the output of ``check_mute_command``.",
```

Comment:
Can you make it clear that, if this string is matched, then the volume is treated as unmuted (assuming I've got that right)?